windows/src/desktop/kmshell/main/OnlineUpdateCheck.pas

```diff
@@ -98,23 +102,141 @@   TOnlineUpdateCheck = class
 
     FShowErrors: Boolean;
     FDownload: TOnlineUpdateCheckDownloadParams;
-
+    {
+      Sets the download path and then calls the DoDownloadUpdates function. If
```

Comment:
I think we need to be very clear on the elevation status for the download process. I believe from reading the code that it is elevated, right -- and that's how it should be, to avoid leaks across the elevation boundary. But we should document this.